src/System.Text.Json/src/System/Text/Json/Serialization/JsonSerializerOptions.cs

```diff
@@ -223,14 +227,16 @@ public bool PropertyNameCaseInsensitive
 
         /// <summary>
         /// Defines how the comments are handled during deserialization.
-        /// By default <exception cref="JsonException"/> is thrown if a comment is encountered.
         /// </summary>
         /// <exception cref="InvalidOperationException">
         /// Thrown if this property is set after serialization or deserialization has occurred.
         /// </exception>
         /// <exception cref="ArgumentOutOfRangeException">
         /// Thrown when the comment handling enum is set to a value that is not supported (or not within the <see cref="JsonCommentHandling"/> enum range).
         /// </exception>
+        /// <remarks>
+        /// By default <exception cref="JsonException"/> is thrown if a comment is encountered.
```

Comment:
"is thrown"... from where?  To me at least it sounds like this is saying that this property throws an exception.

Reply:
Will address the comment feedback separately in the interest of getting the breaking changes in.